lht/src/LHT/Build.hs

```diff
@@ -31,7 +32,8 @@ newtype TargetName = TargetName Text
 data Flag = Flag PackageName Text
 
 data Config = Config
-  { executablePath :: Path.RelFile
+  { cbtBuildDefinition :: CBT.BuildDefinition
+  , executablePath :: Path.RelFile
```

Comment:
__suggestion (nitpick, non-blocking):__ The alignment looks a bit off between the `::`. 

On Pact did you use ormolu or brittany to reformat code? I don't much care which we use, but I'd support using *some* tool now, even if we only agree with 80% of the choices. We can always tweak it later and do global fixup commits to adjust the code it reformats.